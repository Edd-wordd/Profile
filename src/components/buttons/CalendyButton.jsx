import React from 'react'
import { PopupButton } from 'react-calendly'
import { useStyles } from '../styles/buttons/CalendyButton.styles'

function CalendyButton(props) {
  const classes = useStyles(props)

  return (
    <div>
      <PopupButton
        url="https://calendly.com/mona-tech"
        text="Schedule time "
        rootElement={document.getElementById('root')}
        className={classes.calendlyBtn}
      />
    </div>
  )
}

export default CalendyButton
