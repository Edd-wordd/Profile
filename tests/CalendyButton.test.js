import React from 'react'
import testComponent from './testComponent'
import CalendyButton from '../src/components/buttons/CalendyButton'

testComponent(<CalendyButton />, 'CalendyButton')
