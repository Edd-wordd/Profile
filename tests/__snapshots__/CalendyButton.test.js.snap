// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendyButton should render CalendyButton 1`] = `"Component"`;
